### DISCUSSION TITLE
Is it possible to disable allerts on pull request from `github-code-scanning` bot

### DISCUSSION THREAD

**Discussion by jamacku** [community/community#33035] [Code Security]
Is there a way how to disable alerts on pull requests from the `github-code-scanning` bot described in [Viewing an alert on your pull request](https://docs.github.com/en/code-security/code-scanning/automatically-scanning-your-code-for-vulnerabilities-and-errors/triaging-code-scanning-alerts-in-pull-requests#viewing-an-alert-on-your-pull-request)

Thank you in advance for your answers.

**Comment by AlonaHlobina** [comment]
Hi @jamacku! Thank you for your question. 
Unfortunately, there is no possibility to disable alerts from the `github-code-scanning` on pull requests.
May I ask the reason you want to have this functionality?

**Reply by jamacku** [reply]
Hi @AlonaHlobina. Thank you for reaching out.

I'm using `github-code-scanning` feature in my [custom GitHub Action - Differential ShellCheck](https://github.com/redhat-plumbers-in-action/differential-shellcheck). And I have got several requests/questions about the possibility of disabling comments on PR from the `github-code-scanning` bot. Usually, the reasoning for disabling comments was that such comments could be a bit distracting or the bot could create unreasonable noise.

For example, you can see this PR: 

- https://github.com/util-linux/util-linux/pull/1789

It would be nice to have an option to disable comments from the `github-code-scanning` bot on the repository level.
